tests/test_forecast_compiler.py

```diff
@@ -125,3 +125,35 @@ def logger_func(message):
     with pytest.raises(Exception, match="FAIL: Forecast has critical fluctuations"):
         validate_forecast(national_forecast_values,
                           national_capacity, logger_func)
+
+
+def test_validate_forecast_sun_elevation_check():
+    """
+    Test that validate_forecast raises an Exception when forecast values 
+    are ≤ 0 while sun elevation is above SUN_ELEVATION_LOWER_LIMIT.
+    """
+    # Get Sun Elevation Limit value from the environment variable (default 10)
+    sun_elevation_limit = float(os.getenv("SUN_ELEVATION_LOWER_LIMIT", 10))
```

Comment:
does this mean to be FORECAST_VALIDATE_SUN_ELEVATION_LOWER_LIMIT

Reply:
ah could you add this env var in the readme.md. You might need to pull latest branch in? Thanks so much